README.md

```diff
@@ -82,11 +82,26 @@ sudo chmod +x /usr/local/bin/ani-cli
 
 *Note that mpv installed through flatpak is not compatible*
 
+### Mac OS
+
+Install dependencies [(See below)](#Dependencies)
+
+Install [HomeBrew](https://docs.brew.sh/Installation) if not installed.
+
+```sh
+bash
```

Comment:
is bash really required here?

Reply:
someone [like me] might use non-posix compliant shell like fish as their default; might result in unexpected/most likely behavior;